tutorials/metrics.md

```diff
@@ -39,22 +39,34 @@ metrics_key = "/path/to/key.pem"
+| `crio_image_pulls_failures`                      | `name`, `error`                                                                                                                         | Counter   | (DEPRECATED) Failed image pulls by image name and their error category.                                           |
+| `crio_image_pulls_layer_size_{sum,count,bucket}` | buckets in byte for layer sizes of 1 KiB, 1 MiB, 10 MiB, 50 MiB, 100 MiB, 200 MiB, 300 MiB, 400 MiB, 500 MiB, 1 GiB, 10 GiB             | Histogram | Bytes transferred by CRI-O image pulls per layer.                                                                 |
+| `crio_image_layer_reuse`                         | `name`                                                                                                                                  | Counter   | (DEPRECATED) Reused (not pulled) local image layer count by name.                                                 |
+| `crio_containers_oom_total`                      |                                                                                                                                         | Counter   | Total number of containers killed because they ran out of memory (OOM)                                            |
+| `crio_containers_oom`                            | `name`                                                                                                                                  | Counter   | (DEPRECATED) Containers killed because they ran out of memory (OOM) by their name                                 |
+| `crio_processes_defunct`                         |                                                                                                                                         | Gauge     | Total number of defunct processes in the node                                                                     |
+| `crio_operations_total`                          | every CRI-O RPC\*                                                                                                                       | Counter   | Cumulative number of CRI-O operations by operation type.                                                          |
+| `crio_operations_latency_seconds_total`          | every CRI-O RPC\*,<br><br>`network_setup_pod` (CNI pod network setup time),<br><br>`network_setup_overall` (Overall network setup time) | Summary   | Latency in seconds of CRI-O operations. Split-up by operation type.                                               |
+| `crio_operations_latency_seconds`                | every CRI-O RPC\*                                                                                                                       | Gauge     | Latency in microseconds of individual CRI calls for CRI-O operations. Broken down by operation type.              |
```

Comment:
It says microseconds in the description but should be seconds.
I'm also not sure what this is capturing. The latency of the last CRI call? That seems odd. The average? That would be besser calculated from crio_operations_latency_seconds_total.

Reply:
the latency of the last cri call as of now. Then a user can determine the average for the time window they care about (did some operation increase the time a certain amount?)

Reply:
Updated the doc in latest commit.